src/components/organisms/AdminVenueView/components/Timing/Timing.scss

```diff
@@ -33,9 +33,13 @@ $left-section-width: 360px;
   }
```

Comment:
`.Timing` could probably use

`align-items: start;`

so the events table doesn't leave wide gap from the top of its cell.

Reply:
nice catch! fixed in ecdf3dcd597340a1deea0d0f04f0bc2037c2dc65